bucket/dcmtk.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "3.6.7",
+    "description": "DCMTK is a collection of libraries and applications implementing large parts the DICOM standard.",
+    "homepage": "https://dcmtk.org/en/dcmtk/",
+    "license": "BSD-3-Clause",
+    "env_add_path": "./bin/",
```

Comment:
Looks good to me, could you just move `env_add_path` to after `architecture` but before `checkver`? (As per the order here https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md#for-scoop-buckets)

Reply:
Certainly I will update the ordering of the fields. I did initially look to follow the ordering from that documentation however I would note that `architecture` does not currently appear in the preferred ordering listed on that wiki though it is apparent on the linked template. Could the wiki be updated to make it more explicit where `architecture` should belong?

Reply:
Yes that's a good point, I'll make a change to that help page.

Reply:
Thank you @tech189!